src/lib.rs

```diff
@@ -75,10 +75,14 @@ mod engine;
 mod error;
 mod fn_call;
 mod fn_func;
-mod fn_native;
+pub mod fn_native;
 mod fn_register;
 mod module;
 mod optimize;
+#[cfg(not(feature = "no_module"))]
```

Comment:
Not sure if you'd need to expose the entire `plugin` module.  Better to selectively use `pub use` to expose certain subset.

Reply:
The comment above on the other `pub mod` also applies here.